tests/generators/OperationGenerator.ts

```diff
@@ -242,4 +216,22 @@ export default class OperationGenerator {
     const operationJws = await OperationGenerator.createOperationJws(OperationType.Recover, payload, existingRecoveryKeyId, existingRecoveryPrivateKey);
     return Buffer.from(JSON.stringify(operationJws));
   }
+
+  /**
+   * Generates a single element array with a identity hub service object for DID document
+   * @param instances the instance field in serviceEndpoint. A list of DIDs
+   */
+  public static createIdentityHubServiceEndpoints (instances: string[]): any[] {
+    return [
+      {
+        'id': 'IdentityHub',
+        'type': 'IdentityHub',
+        'serviceEndpoint': {
+          '@context': 'schema.identity.foundation/hub',
+          '@type': 'UserServiceEndpoint',
+          'instance': instances
```

Comment:
Looks like we are only creating the 'UserServiceEndpoint' (see ````'@type': 'UserServiceEndpoint'````), can we change the function name to reflect that please? 

Also, seems like that the 'instance' property is supposed to be single-valued right? If single valued, then let's change the input to ````instance: string````

Reply:
I will double check with the rest of the team to make sure if it's supposed to be singular or plural.

Reply:
I changed it to instances for now and everything seems to be working fine.

Reply:
Changed instances and related validations.